### PR TITLE
Fixed test cases and bootstrap

### PR DESCRIPTION
**Pull request by altmannmarcelo**
Adjusted keyring to always pass plugin dir.
Bootstrap to download correct glibc version

### CODE REVIEW

File: storage/innobase/xtrabackup/test/bootstrap.sh [hunk 1]
```diff
@@ -31,6 +31,20 @@ function ssl_version() {
     echo ${sslv}
 }
 
+function glibc_version() {
+    glibc=$(ldd --version | head -1 | awk '{print $NF}')
+    case ${glibc} in
+        2.12|2.17|2.27|2.28|2.31|2.34|2.35) ;;
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/test/bootstrap.sh:35]: how do we know this is valid list of glibc versions?

File: storage/innobase/xtrabackup/test/bootstrap.sh [hunk 2]
```diff
@@ -31,6 +31,20 @@ function ssl_version() {
...
+        *)
+            if ! test -r "${1}"; then
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/test/bootstrap.sh]: where do we get ${1} from?

File: storage/innobase/xtrabackup/test/bootstrap.sh (hunk 1 above)
Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/test/bootstrap.sh:35]: This is the list from which we produce binaries from.

File: storage/innobase/xtrabackup/test/bootstrap.sh (hunk 2 above)
Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/test/bootstrap.sh]: I think this all originated from https://github.com/percona/percona-xtrabackup/commit/6d0b9ea4172cb4b4c738838591444c2206dd858e , and I think this commit got the `if ! test -r "${1}"; then` from the main loop. In the SSL and GLIBC context this is not required. I will remove

**Review by satya-bodapati — APPROVED** [review]
please test and merge

**Comment by altmannmarcelo** [comment]
@satya-bodapati all tests are now passing.

https://pxb.cd.percona.com/job/percona-xtrabackup-8.0-test-param/367/#showFailuresLink

With this I discovered https://jira.percona.com/browse/PXB-3173 :tada:

**Pull request merged**